contracts/DomainRegistry.sol

```diff
@@ -55,25 +58,35 @@ contract DomainRegistry is Initializable, OwnableUpgradeable {
     event DomainReleased(address indexed controller, string domainName);
 
     uint256 public constant REWARD_PERCENT_OWNER = 10;
+    IERC20 public token;
+    AggregatorV3Interface internal priceFeed;
 
     /// @notice Reinitializes the contract and sets domain level prices
-    function reinitialize() public reinitializer(2) {
+    function initialize() initializer public {
         __Ownable_init(msg.sender);
 
-        setDomainLevelPrice(1, 0.75 ether);
-        setDomainLevelPrice(2, 0.5 ether);
-        setDomainLevelPrice(3, 0.25 ether);
-        setDomainLevelPrice(4, 0.15 ether);
-        setDomainLevelPrice(5, 0.1 ether);
+        token = IERC20(msg.sender);
```

Comment:
This is absolutely wrong. This is not the token contract, which this` initialize() `method
     -5 point